test/education.e2e-spec.ts

```diff
@@ -131,7 +131,9 @@ describe('Education e2e', () => {
       }
     );
 
-    expect(educations.items.length).toBeGreaterThanOrEqual(numEducations);
+    expect(result.user.education.items.length).toBeGreaterThanOrEqual(
+      numEducations
+    );
```

Comment:
Haha I'm dumb 🤦 . I got that change on one of the suites, but missed it on the rest.